test/forge/blade/generateMsgBridgeStorage.ts

```diff
@@ -21,6 +24,29 @@ let blockHash: any;
 let currentValidatorSetHash: any;
 let bitmaps: any[] = [];
 let aggVotingPowers: any[] = [];
+let msgs = [
+  {
+    id: 1,
+    sourceChainId: sourceChainId,
+    destinationChainId: destinationChainId,
+    sender: ethers.constants.AddressZero,
+    receiver: ethers.constants.AddressZero,
```

Comment:
It would be awesome if some of the test messages would contain some dummy (non-zero) addresses and hashes. :slightly_smiling_face: